### PR TITLE
PIM-8496: new form of acceptance tests

### PR DESCRIPTION
**Pull request by juliensnz**
**Description (for Contributor and Core Developer)**

In this PR I fixed the renderView method to make it work as expected (it was getting a parameter but was not using it and was also rendering views in a weird way). I also added minimal css to our test webpack build to have nice tests (with the new styled components, the tests no have css and look exactly the same as in the real UI. It will help us having more stable tests and also closer to the real result.

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added legacy Behats               | Todo
| Added acceptance tests            | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: tests/front/acceptance/cucumber/tools.js [hunk 1]
```diff
@@ -54,16 +54,20 @@ const convertItemTable = dataTable => {
...
 
-      return form;
-    });
-  }, data);
+        return form;
+      });
+    },
+    data,
+    extension
+  );
```
Comment (grena) [inline tests/front/acceptance/cucumber/tools.js]: I don't understand the changes in this file

**Comment by SamirBoulil** [comment]
Can you explain in 2 points what is done in the changes ?

File: tests/front/acceptance/cucumber/tools.js (hunk 1 above)
Comment (juliensnz) [reply tests/front/acceptance/cucumber/tools.js]: this method was previously not configurable at all. It was supposed to use the `extension` variable but used a fixed string instead `pim-catalog-volume-index`. So I made the method work as expected and also changed a bit the way to render the form element to be less error prone :)

**Comment by juliensnz** [comment]
@SamirBoulil you are right, sorry about that, I added description in the PR

File: tests/front/acceptance/cucumber/tools.js (hunk 1 above)
Comment (grena) [reply tests/front/acceptance/cucumber/tools.js]: Thanks for the explanation :)

**Pull request merged**